### PR TITLE
New images on customize page + new open/close docs

### PR DESCRIPTION
**Pull request by captainbrosset**
We have various blog posts coming out that mention the ability to open and close tabs in Edge DevTools. So I'm taking this opportunity to add quick docs about this in the "Customize Microsoft Edge DevTools" page.

At the same time, I'm also updating the screenshots to match the new Fluent icons and be in light theme.

I also removed an outdated information that was mentioning the need to enable an experimental flag, which is not needed anymore.

* Old page: https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/customize/
* New page: https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/customize/?branch=pr-en-us-1436

### CODE REVIEW

**Review by hxlnt — APPROVED** [review]

File: microsoft-edge/devtools-guide-chromium/customize/index.md [hunk 1]
```diff
@@ -66,19 +61,32 @@ You can move tools between the main panel and the drawer.
...
 
+## Open and close panels  
+
+Many of the available panels in DevTools are not opened by default to avoid cluttering the interface with rarely used tools. To open a tool in the main panel or the drawer, select the plus button \(![More Tools](../media/open-tab-icon.png)\) to the right of the tabs and choose a tool from the list.  
+
+:::image type="complex" source="../media/open-tool-in-main-panel-or-drawer.png" alt-text="The + button to open a new tool" lightbox="../media/open-tool-in-main-panel-or-drawer.png":::
+   The `+` button to open a new tool  
+:::image-end:::  
+
+Select the close button \(![Close Tool](../media/close-tab-icon.png)\) in a tab to close the associated tool.
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/customize/index.md:84]:
```suggestion
To close a tool, select the **Close** \(![Close Tool](../media/close-tab-icon.png)\) button in the tool's tab.
```

File: microsoft-edge/devtools-guide-chromium/customize/index.md [hunk 2]
```diff
@@ -29,9 +29,10 @@ This page lists the ways to customize Microsoft Edge DevTools.
 
 **Settings** > **Preferences** contains many options for customizing DevTools.
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/customize/index.md:31]: Document the button:
Settings > Preferences > Restore defaults and refresh

File: microsoft-edge/devtools-guide-chromium/customize/index.md [hunk 3]
```diff
@@ -66,19 +61,32 @@ You can move tools between the main panel and the drawer.
...
+   The `+` button to open a new tool  
+:::image-end:::  
+
+Select the close button \(![Close Tool](../media/close-tab-icon.png)\) in a tab to close the associated tool.  
+
+:::image type="complex" source="../media/close-tool-in-main-panel-or-drawer.png" alt-text="The tab's close button" lightbox="../media/close-tool-in-main-panel-or-drawer.png":::
+   The tab's close button  
+:::image-end:::  
+
 ## Change DevTools placement
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/customize/index.md:91]: Change heading from "Theme" to "Color themes".

File: microsoft-edge/devtools-guide-chromium/customize/index.md [hunk 4]
```diff
@@ -66,19 +61,32 @@ You can move tools between the main panel and the drawer.
...
 
 > [!NOTE]
-> By default, the **Network** tool is usually the fourth from the left.  In the following figure, the **Network** tool is the first from the left.  
+> By default, the **Network** tool is usually the fifth from the left.  In the following figure, the **Network** tool is being moved to the first position from the left.  
 
 :::image type="complex" source="../media/customize-network-first-position.msft.png" alt-text="Custom order of Devtools in a panel" lightbox="../media/customize-network-first-position.msft.png":::
    Custom order of Devtools in a panel  
 :::image-end:::  
 
+## Open and close panels
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/customize/index.md:76]: Consider retitling to "Open and close tools".  Give hints or explicit discussion, that tab ~= tool ~= panel (~= pane).  A tab (as in, **More tabs** (>>)) is a common, concrete UI item meaning both the little rect & pane.  In the Command Menu, tools are called _panels_.  Practically, users need to know both terms.

File: microsoft-edge/devtools-guide-chromium/customize/index.md [hunk 5]
```diff
@@ -66,19 +61,32 @@ You can move tools between the main panel and the drawer.
     :::image type="complex" source="../media/move-to-drawer.msft.png" alt-text="Move tool from the main panel to the Drawer" lightbox="../media/move-to-drawer.msft.png":::
        Move tool from the main panel to the **Drawer**
     :::image-end:::  
-    
 
 ## Reorder panels
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/customize/index.md:65]:
```suggestion
## Reorder tools
```
Compare https://developer.chrome.com/docs/devtools/customize/#reorder

File: microsoft-edge/devtools-guide-chromium/customize/index.md (hunk 2 above)
Comment (captainbrosset) [reply microsoft-edge/devtools-guide-chromium/customize/index.md:31]: Done

**Review by mikehoffms — APPROVED** [review]

**Pull request merged**